app/code/Magento/Customer/Setup/CustomerSetup.php

```diff
@@ -234,7 +257,7 @@ public function getDefaultEntities()
                         'label' => 'Date of Birth',
                         'input' => 'date',
                         'frontend' => \Magento\Eav\Model\Entity\Attribute\Frontend\Datetime::class,
```

Comment:
why don't we simply this?

Reply:
@Usik2203 could you please import this one as well?
Thanks.